src/components/Header.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {Nav, NavItem, NavLink} from 'reactstrap'
+const Header = () => {
+  return (
+    <>
+    <Nav
```

Comment:
Check indentation in this component as well and fill with content you want your user to see in the Header.